schemas/rum/view-schema.json

```diff
@@ -103,6 +103,12 @@
               "minimum": 0,
               "readOnly": true
             },
+            "time_to_first_byte": {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
            "first_byte": {
```
* matches the resource naming
* matches other view namings where we don't use `time_to_...` (ex: we could've used `time_to_load_event` but we didn't)

Reply:
ok sounds good to me it also match the original backend.
I'll do the change, cc @JoelPfaffDD

Reply:
Please disregard the original backend naming choice I have taken, I needed a name to start the dev, and it is easy to change there.
Just consider the proposal above on its own merit, since this one cannot be changed easily anymore.

Reply:
sure, @BenoitZugmeyer' arguments were enough for me 🙂